subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -786,8 +813,15 @@ public BuildOperationDescriptor.Builder description() {
         });
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @implNote Usage: This method should only be called on resolvable configurations and will throw an exception if
```

Comment:
How come this doc is included in the implementation and not the interface? I feel like this javadoc is only really necessary in the interface, since it is pretty clear that this is a resolvable-only method based on the first line of the method.